internal/pkg/deploy/cloudformation/stack/transformers_test.go

```diff
@@ -1626,7 +1665,7 @@ func Test_convertSubscribe(t *testing.T) {
 		},
 		"topic not allowed": {
 			inSubscribe: &manifest.SubscribeConfig{
-				Topics: &[]manifest.TopicSubscription{
+				Topics: []manifest.TopicSubscription{
```

Comment:
Can we add a separate unit test validating that a maliciously constructed input fails?

Maliciously constructed meaning it contains the correct elements (service and topic name) with extra info which causes it to point to a different topic.